README.md

```diff
@@ -1,24 +1,24 @@
-[![Build Status](https://travis-ci.org/adrcotfas/Goodtime.svg?branch=master)](https://travis-ci.org/goodtime-productivity/Goodtime) [![Crowdin](https://d322cqt584bo4o.cloudfront.net/goodtime/localized.svg)](https://crowdin.com/project/goodtime) 
+[![Build Status](https://travis-ci.org/adrcotfas/Goodtime.svg?branch=master)](https://travis-ci.org/goodtime-productivity/Goodtime) [![Crowdin](https://d322cqt584bo4o.cloudfront.net/goodtime/localized.svg)](https://crowdin.com/project/goodtime)
 
 # Goodtime
-
-<a href='https://play.google.com/store/apps/details?id=com.apps.adrcotfas.goodtime&utm_source=global_co&utm_medium=prtnr&utm_content=Mar2515&utm_campaign=PartBadge&pcampaignid=MKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1'><img alt='Get it on Google Play' src='https://play.google.com/intl/en_us/badges/images/apps/en-play-badge-border.png' height="80" /></a>
-<a href="https://f-droid.org/en/packages/com.apps.adrcotfas.goodtime">
-    <img src="http://i.imgur.com/baSPE7X.png"
-         alt="Get it on F-Droid" height="80">
-</a>
+[<img src="https://play.google.com/intl/en_us/badges/static/images/badges/en_badge_web_generic.png"
+    alt="Get it on Google Play"
+    height="80">](https://play.google.com/store/apps/details?id=com.apps.adrcotfas.goodtime&utm_source=global_co&utm_medium=prtnr&utm_content=Mar2515&utm_campaign=PartBadge&pcampaignid=MKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1)
+[<img src="https://fdroid.gitlab.io/artwork/badge/get-it-on.png"
+    alt="Get it on F-Droid"
+    height="80">](https://f-droid.org/packages/com.apps.adrcotfas.goodtime)
 
 A minimalist but powerful productivity timer designed to keep you focused and free of distractions.
-<br>Alternate between focused work sessions and short breaks handled by the app.
-<br>Get rid of procrastination and improve your time management!
+Alternate between focused work sessions and short breaks, all handled by the app.
+Get rid of procrastination and improve your time management!
 
-<br>⏳ split your work into sessions of focus and then reward yourself with a break
-<br>☕ after more sessions of work, take a longer break
+⏳ split your work into focused sessions and then reward yourself with a break
+☕ after mulitple work sessions, take a longer break
 
-**Features**
+## Features
 - No ads
 - No tracking or personal data collection
-- Open-source
+- Free & Open Source
```

Comment:
I would not add the "Free" part since I'm using the same repo for both Google Play(in app purchase to unlock some features) and F-Droid(fully unlocked) flavors.

Reply:
In this case "Free and Open Source" is a whole (FOSS), it's just a way to Say it's Free Software (cf. Free Software Foundation) and Open Source Software (cf. Open Source Initiative).